dockerfiles/tasks.py

```diff
@@ -81,12 +81,17 @@ def shell(c, running=True, container='web'):
 @task(help={
     'command': 'Command to pass directly to "django-admin" inside the container',
     'running': 'Execute "django-admin" in a running container',
+    'backupdb': 'Backup postgres database when running Django "migrate" command',
 })
-def manage(c, command, running=True):
+def manage(c, command, running=True, backupdb=False):
     """Run manage.py with a specific command."""
     subcmd = 'run --rm'
     if running:
         subcmd = 'exec'
+
+    if command.startswith('migrate') and backupdb:
```

Comment:
Is there a reason to check for `migrate` here? We can backup the DB on another command if we want.

Reply:
Yes. Originally I used `backupdb=True` as default because I wanted to backup every time we run `migrate`. Now that the default is `False`, we can backup on every `manage` command if we want.

We can just remove the `migrate` from here and keep the default as it is now: `False`. Yes.

Reply:
I removed the `.startswith('migrate')` in a new commit.